packages/vscode-extension/src/extension.ts

```diff
@@ -88,7 +86,18 @@ export async function activate(context: ExtensionContext) {
   commands.executeCommand("setContext", "RNIDE.sidePanelIsClosed", false);
 
   async function showIDEPanel(fileName?: string, lineNumber?: number) {
-    commands.executeCommand("setContext", "RNIDE.sidePanelIsClosed", false);
+    await commands.executeCommand("setContext", "RNIDE.sidePanelIsClosed", false);
+
+    if (Platform.OS === "macos") {
+      try {
+        await setupPathEnv();
```

Comment:
this isn't the right place do do this, as when side panel mode is on, this will be called too late. We should move this to extension activation